Source/XUnitExtensions/Lit/ExitCommand.cs

```diff
@@ -1,22 +1,32 @@
+using System.ComponentModel;
 using System.IO;
 using System.Threading.Tasks;
 
 namespace XUnitExtensions.Lit;
 
 
 public class ExitCommand : ILitCommand {
-  private readonly int expectedExitCode;
+  private readonly string expectedExitCode;
   private readonly ILitCommand operand;
 
-  public ExitCommand(int exitCode, ILitCommand operand) {
+  public ExitCommand(string exitCode, ILitCommand operand) {
     this.expectedExitCode = exitCode;
     this.operand = operand;
   }
 
   public async Task<int> Execute(TextReader inputReader,
     TextWriter outputWriter, TextWriter errorWriter) {
-    var exitCode = await operand.Execute(inputReader, outputWriter, errorWriter);
-    if (exitCode == expectedExitCode) {
+    var exitCode = 1;
+    try {
+      exitCode = await operand.Execute(inputReader, outputWriter, errorWriter);
+    } catch (Win32Exception) {
```

Comment:
I suspect this exception would be a different one on non-Windows platforms. Or is it used on all platforms in .NET?

Reply:
Windows does not have the "rm" command so I'm not sure if the original returns an exception if the file does not exist or not.

Reply:
Note: the code was working fine in other platforms without the try catch, so I'm not expecting this to break any other platform.